.circleci/tests/test_ALFF.py

```diff
@@ -21,6 +21,8 @@ def test_nifti_alff(data_dir, tmp_path_factory):
     has increased.
     """
     # Get the file names
+    data_dir = os.path.join(data_dir,
+                            "data/fmriprepwithfreesurfer")
```

Comment:
Can you append `data/fmriprepwithfreesurfer` to the data_dir that's fed into the pytest call instead?

Reply:
I think the problem was that we used both nifti with and without freesurfer

Reply:
In that case, what about including `data` in data_dir, so you only need to add the dataset name in the test functions?

Reply:
can have a shot at that right now

Reply:
This looks like it's working